drake/systems/plants/RigidBody.cpp

```diff
@@ -26,7 +32,14 @@ const DrakeJoint& RigidBody::getJoint() const {
   if (joint) {
     return (*joint);
   } else {
-    throw runtime_error("Joint is not initialized");
+    if (linkname == std::string(kWorldLinkName)) {
```

Comment:
It shouldn't be necessary to construct a `std::string` temporary from `kWorldLinkName`, right?